COMPARE.md

```diff
@@ -0,0 +1,54 @@
+# Comparing Standard to X
+
+_Where appropriate, we compare with `divnix/paisano`, instead_.
+
+## Comparison with tools in the Nix ecosystem
+
+### flake-utils
+
+`numtide/flake-utils` is a small & lightweight utility with a focus on generating flake file _outputs_ in accordance with the packaging use case built-in into the Nix CLI tooling.
+
+Paisano, in turn, is an importer with a focus on _code organization_; it still plugs well into a `flake.nix` file, but also preserves its index function by keeping it clean.
+While you _can_ use it to match the schema that Nix CLI expects, it also enables more flexibility as it is not specially optimized for the Nix _package manager use case_.
+
+### flake-parts
+
+`hercules-ci/flake-parts` is a component aggregator with a focus on a flake schema that is built-in into the Nix CLI tooling that makes use of the NixOS module system for composability and aggregation.
+
+Paisano, in turn, is an importer with a focus on _code organization_; it still plugs well into a `flake.nix` file, but also preserves its index function by keeping it clean.
+While you _can_ use it to match the schema that Nix CLI expects, it also enables more flexibility as it is not specially optimized for the Nix _package manager use case_.
+
+To a lesser extent, Paisano is also a component aggregator for your flake outputs.
+However, in that role, it gives you back the freedom to use the output schema that best fits your problem domain.
+
+Convergence towards the Flakes output schema is provided via the harvester family of functions (`winnow`, `harvest` & `pick`).
+Depending on the domain schema, it can be a _lossy_ convergence due the lesser expressivity of the flake output schema.
+
+Flake Parts aggregates bespoke use-case specific interfaces implemented in the module system DSL.
+Paisano, in turn, focuses on code organization along high level code boundaries connected by generic interfaces.
+
+The core tenet of Flake Parts is bespoke module system DSL interfaces for each use case.
```

Comment:
This sentence is hard to parse, specifically "bespoke module system DSL interfaces". There must be a cleaner way to say this, but nothing comes to mind atm.